spec/02-integration/09-hybrid_mode/01-sync_spec.lua

```diff
@@ -59,13 +63,18 @@ for _, strategy in helpers.each_strategy() do
       local route_id
 
       it("proxy on DP follows CP config #flaky", function()
```

Comment:
if we're re-enabling these should the flaky be dropped?

Reply:
Good catch! the other similar file already had #flaky removed. Force-pushing the removal to try it on CI.